plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -232,6 +232,64 @@ public void testShowColumns()
         assertEquals(expected, actual);
     }
 
+    @Test
+    public void testNullPredicate()
```

Comment:
the error-proneness is generic (it's not the first time we forget about `domain.isNullAllowed()`) and so we should have a generic test that covers all the the connectors.

do you happen to know why

https://github.com/trinodb/trino/blob/36cada014c8cd1a227920a5508cb68174175f779/testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java#L1226-L1227

and

https://github.com/trinodb/trino/blob/36cada014c8cd1a227920a5508cb68174175f779/testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java#L1234-L1238

did not let us uncover the bug earlier?

Reply:
I guess this tests runs only if the connector has support for creating tables. And elasticsearch doesn't support it as of now so it got missed.

Reply:
I think we can have an extra extension point to inject the data via non SQL methods and base connector tests could have an impl for it

Reply:
Exactly. These tests got missed because elasticsearch tests has disabled the support for create table.

Reply:
Thanks, i missed that.